services/skus/README.md

```diff
@@ -5,19 +5,35 @@
 
 1. Begin with development setup steps 1 through 4 from the [bat-go readme](https://github.com/brave-intl/bat-go/blob/master/README.md)
 
-2. Bring up the SKUs containers locally with ```make docker-refresh-payment```
+2. Bring up a dev environment with `make docker-dev`
```

Comment:
suggestion: we could change this too 

2. To bring up an environment to exercise the API run `make docker-up-dev` once the containers have started the API will be available at `localhost:3333` for example, executing `curl localhost:3333/health-check` will provide a health check

Then remove steps 3 and 4